### PR TITLE
Re write some of the README

### PR DESCRIPTION
**Pull request by iphands**
* You dont need to have your repo in GOHOME thanks to go modules
* You dont need to clone the middlewares project (go get would have worked, but go modules solves this)
* The documentation for the FF stuff and how to obtain the certs is duplicated from https://mojo.redhat.com/docs/DOC-1144091
* Getting developer certs does not let you talk to subscription.api.redhat.com use the QA host in the docs
* Store the personal developer config inside a folder that is *not* tracked by Git

### CODE REVIEW

**Comment by kruai** [comment]
I said to clone in GOHOME because, at least before the modules, the project wouldn't work anywhere else. I'll attempt re-cloning in my usual workspace and see if it works for me. Once I confirm, I'll approve this PR

File: README.md [hunk 1]
```diff
@@ -27,31 +22,20 @@ sudo sh /scripts/dev_deps.sh
...
-* From here you can export your crt and key like so:
+* You can export your crt and key like so:
     `openssl pkcs12 -in your-p12-cert.p12 -out your-key.key -nocerts -nodes`
     `openssl pkcs12 -in your-p12-cert.p12 -out your-cert-sans-key.crt -clcerts -nokeys`
 
 ## Create your config file
 
-You'll need to make a config file specific to your machine. Create a file `/config/development.{your_user_name}.sh` and enter the following:
+You'll need to make a config file specific to your machine.
+Create a file `mkdir -p ./local && ./local/development.env.sh` and enter the following:
```
Comment (kruai) [inline README.md]: You sure this syntax is right? I feel like it should be `mkdir -p ./local && touch ./local/development.env.sh` or something like that

Comment (iphands) [reply README.md]: Yeah I like it... gonna change

**Comment by ryelo** [comment]
@iphands A few comments on my end from a MacOS standpoint:

1. We use `brew install go` to install instead of `dnf`
2. When we run the go app, we use `sudo`
`sudo sh scripts/watch.sh local/development.env.sh`

**Comment by iphands** [comment]
> @iphands A few comments on my end from a MacOS standpoint:
> 
> 1. We use `brew install go` to install instead of `dnf`
> 2. When we run the go app, we use `sudo`
>    `sudo sh scripts/watch.sh local/development.env.sh`

Wait I get brew vs sudo ... but why are you running as root.
I dont think we need that or should advocate that others do it.

**Pull request merged**